src/lib/y2network/dialogs/s390_device_activation.rb

```diff
@@ -88,7 +88,7 @@ def run
           if configured
             interface_name = activator.configured_interface
             builder.name = interface_name
-            add_interface(interface_name)
+            online!(builder)
```

Comment:
Please, do not use an exclamation mark unless you have another method with no exclamation (`sub` and `sub!` is OK, for instance).

Reply:
Actually, `add_interface` and `online!` are private methods that are called only once. And `add_interface` is an one-liner. Why not merging both of them? You could even add an `online:` argument if needed.

Reply:
I will rename it to refresh_devices. About the add_interface, I maintained the convenience method because it was already mocked in the unit test, but will merge both and adapt the test.